### DISCUSSION TITLE
Add docker-compose to the CasaOS UI

### DISCUSSION THREAD

**Discussion by Brottus** [IceWhaleTech/CasaOS#223] [Ideas] [answered]
#### Is your idea related to a problem? Please describe:
[Docker Site](https://docs.docker.com/compose/gettingstarted/#:~:text=Get%20started%20with%20Docker%20Compose%201%20Setup%20%F0%9F%94%97.,Update%20the%20application%20%F0%9F%94%97.%20...%20More%20items...%20)
So the issue as of now is that you have to create the containers one by one. With docker compose you can easily manage applications which are divided into micro services so its easier to manage one application. So for instance you can easily have an app with an frontend container and a backend container. Like [Wordpress](https://docs.docker.com/samples/wordpress/) 
#### Describe the solution you'd like:
A simple Field where you can copy paste the docker compose file into would be perfect for the start. It then should also be pausible to start and stop the container via the compose file. 
After that a parser like with the config fields for the container would be top notch so you can really easy manage everything

#### Describe alternatives you've considered:
You can have the same effect with adding container one by one and add create a docker network for them. But that is time consuming and you don't have the whole config visible. Other than that there is also kubernetes or rancher or so on. But that is probably to much into the future ;-).
#### Additional context
I know that this can be a tedious task but you would have one user who would use it ;-).

**Comment by zarevskaya** [comment]
Hello, docker-compose is implemented recently:

![Capture d’écran 2022-05-23 192922](https://user-images.githubusercontent.com/60230221/169875090-b2c1590f-1d31-4264-8ce0-e926d1ac8230.png)

However in some cases there would be a problem with the database.

I think it is necessary to identify the ip of the database:

https://youtu.be/RPPbixWEXp0

The team is looking at how to properly implement the database

**Reply by toastbrotch** [reply]
this option is a bit hidden:
![2023-06-17 10_26_16-CasaOS – Mozilla Firefox](https://github.com/IceWhaleTech/CasaOS/assets/1016140/096645da-bb14-43fd-8cbb-85b65ee04670)

**Comment by tigerinus** [comment, marked as answer]
Docker Compose is natively supported starting from CasaOS v0.4.4

**Reply by aniel300** [reply]
i literally just googled this, lol.

**Comment by BuriKizilkaya** [comment]
If I have more than one container specified, then just the first container will be imported. Is that true?

**Reply by Dumpling757** [reply]
No it will show both containers:

![image](https://github.com/IceWhaleTech/CasaOS/assets/90244905/2a77bff9-076a-44ed-9c78-12ba5524f867)

I have, however run into an issue that my third container in the yaml file is not shown.
